console/src/components/WeatherItem/WeatherItem.less

```diff
@@ -0,0 +1,33 @@
+@import "~@enact/agate/styles/skin.less";
+
+.weatherItem{
+	&.featured {
+		.item {
+			.text{
+				font-size: 36px;
+			}
+		}
+		.icon {
+			width: 150px;
+			height: 150px;
+			object-fit: cover;
+		}
+	}
+
+	.item {
+		.text {
+			font-size: 28px;
```

Comment:
Let's have this be a variable so themes can choose what size they want it to be. (Same for the other measurements in this file.) Just move the `.applySkins({` up to the top of the rule-set, and the `});` down to the bottom to cover `.icon` too.